src/pages/api/github-load-notes.ts

```diff
@@ -109,6 +113,7 @@ export function parseMdxNotesFile(text: string) {
       return {
         text: contents.trim(),
         type: 'staff',
+        image: eggheadLogo,
```

Comment:
for staff notes we default to egghead logo
![image](https://user-images.githubusercontent.com/25487857/149173127-773427c4-1b1c-4100-af50-06f0f694852e.png)